README.md

```diff
@@ -196,6 +196,10 @@ content: ({tiastId, dismiss}) => (
 
 ![alt tag](https://raw.githubusercontent.com/ealush/butter-toast/master/assets/rec0.gif)
 
+# Universal app build
+To use ButterToast in universal app you should run `npm run build:universal`
```

Comment:
The consumers should not need to build `universal` themselves. The package should come pre-built with `universal` support, and we should just instruct them on how to actually consume the universal version.